src/shared/components/layouts/ReleaseInfo.tsx

```diff
@@ -0,0 +1,34 @@
+import { Method } from 'axios';
+import { Link } from 'react-router-dom';
+import apiUrls from '../../config/apiUrls';
+import useDataApi from '../../hooks/useDataApi';
+import { Namespace } from '../../types/namespaces';
+
+const fetchOptions: { method: Method } = { method: 'HEAD' };
+const ReleaseInfo = () => {
+  // TODO: replace with statistics endpoint
+  const { headers } = useDataApi(
+    `${apiUrls.search(Namespace.uniprotkb)}?query=*&size=0`,
+    fetchOptions
+  );
+  const releaseDate = headers?.['x-release']
+    ? new Date(headers['x-release'])
+    : undefined;
+
+  if (!releaseDate) {
+    return <>Loading release information</>;
```

Comment:
It would be good to have the link to Statistics be visible even without the release information yet as this is not network dependent.
Also, I understand this component has been moved here to reuse it, but while I think it is OK to display "Loading release information" in the footer, I believe it might be a bit disruptive to the user to see this happen in the header.
I would rather reserve the space, and slowly make it appear when we have the information (transition opacity)

Reply:
Yep, it doesn't look great at the moment... I was wondering about using a similar trick as your cursor for "Find my ... " but didn't get a chance to look into it.
Also, how would you go about reserving the space? swap an empty span with as width the expected number of `ch`?

Reply:
Do you have a screenshot of how it looks like at the moment?